src/CreatorTokenSwapRouter.sol

```diff
@@ -18,6 +21,10 @@ contract CreatorTokenSwapRouter {
   // WETH -> USDC, order is switched because of V3_SWAP_EXACT_OUT
   bytes private path;
 
+  /// @dev Contract constructor sets up the Universal Router and token addresses.
```

Comment:
Should be `@notice` instead of `@dev`

Reply:
updated in [e159a53](https://github.com/showtime-xyz/creator-tokens/pull/46/commits/e159a53031291a7ef77626ca2d681aec08053372)